src/Identifier/LdapIdentifier.php

```diff
@@ -92,18 +97,30 @@ public function __construct(array $config = [])
      */
     protected function _checkLdapConfig()
     {
-        if (!isset($this->_config['bindDN'])) {
-            throw new RuntimeException('Config `bindDN` is not set.');
+        if (!isset($this->_config['filter'])) {
+            throw new RuntimeException('Config `filter` is not set.');
         }
-        if (!is_callable($this->_config['bindDN'])) {
+        if (!is_callable($this->_config['filter'])) {
             throw new InvalidArgumentException(sprintf(
-                'The `bindDN` config is not a callable. Got `%s` instead.',
-                gettype($this->_config['bindDN'])
+                'The `filter` config is not a callable. Got `%s` instead.',
+                gettype($this->_config['filter'])
             ));
         }
         if (!isset($this->_config['host'])) {
             throw new RuntimeException('Config `host` is not set.');
         }
+
+        if (!isset($this->_config['bindDN'])) {
+            throw new RuntimeException('Config `bindDN` is not set.');
+        }
+
+        if (!isset($this->_config['bindPassword'])) {
+            throw new RuntimeException('Config `bindPassword` is not set.');
+        }
+
+        if (!isset($this->_config['baseDN'])) {
```

Comment:
Adding required configuration values would break existing usage of this identifier, could we do this in a way that doesn't require people to update their code unless they want to use the new features?